packages/libs/src/services/ethWeb3Manager/EthWeb3Manager.ts

```diff
@@ -50,8 +51,12 @@ export class EthWeb3Manager {
       throw new Error('missing web3Config property: providers')
 
     // MultiProvider implements a web3 provider with fallback.
-    const provider = new MultiProvider(web3Config.providers)
-
+    let provider: MultiProvider | string
+    if (!web3Config.disableMultiProvider) {
+      provider = new MultiProvider(web3Config.providers)
+    } else {
+      provider = web3Config.providers![0]!
```

Comment:
Cool!

Weird that we need the first `!` since we're doing a check for `if (!web3Config.providers)` above 🤔

Reply:
agree :(